R/generate_simulated_data.R

```diff
@@ -405,7 +392,7 @@ generate_simulated_data <- function(r_in_weeks = # nolint
       log_state_rt = log_r_state_week,
       state_deviation_ar_coeff = phi_rt,
       scaling_factor = scaling_factor,
-      sigma_eps = sigma_generalized^(1 / n_sites),
+      sigma_eps = sqrt(sigma_generalized^(1 / n_sites)),
```

Comment:
To prevent confusion, might be nice to declare this above e.g. 
`stdev_generalized = sqrt(sigma_generalized^(1 / n_sites))`
Then
`sigma_eps = stdev_generalized`. 

Alternatively, it might be nice if we called the generalized variance variable `sigma_sq_generalized`...